infrastructure/zk/src/init.ts

```diff
@@ -32,6 +32,7 @@ export async function init(initArgs: InitArgs = DEFAULT_ARGS) {
     let envFileContent = fs.readFileSync(process.env.ENV_FILE!).toString();
     envFileContent += `VALIDIUM_MODE=${validiumMode}\n`;
     fs.writeFileSync(process.env.ENV_FILE!, envFileContent);
+    process.env.VALIDIUM_MODE = validiumMode.toString();
```

Comment:
Is it (and code above) still required given that we now have a proper configuration option?

Reply:
This case is very particular. That line is there only because the L1 contracts (`Executor.sol`) need that variable set before compilation. I really thought about improving here but nothing came to my mind. Let me know if, now that you have a complete context of the design decision, you come up with a new strategy for this.

Reply:
Wouldn't it be better to set this variable in the script that deploys the corresponding contract, and pass it to the script from init via a CLI argument?
Behavior that depends on some environment variable is efficiently a global state, so by passing it via CLI you'll make the process more explicit, and the env variable would only exist in a much more local context.

Reply:
Indeed it is, we'll update it asap.